reports/namespace-usage/README.md

```diff
@@ -1,4 +1,44 @@
 # Namespace Usage
 
-Ouputs a JSON report showing the CPU and memory requested vs. used, for all
-namespaces in the cluster.
+Ouputs a JSON report showing 
+- the CPU requested vs. used, for all namespaces
+- the Memory requested vs. used for all namespaces
+- the number of Pods Hard limits vs. current for all namespaces
+- number of containers for all namespaces
+
+The main package in this report will perform the following steps:
+
+- Fetch the kubeconfig from the s3 bucket 
```

Comment:
I don't know why this bothers me, but can you pick upper case or lowercase for all bullets